.travis.yml

```diff
@@ -19,19 +19,19 @@ jobs:
     - stage: test
       script:
         - ./scalafmt --test
-        - sbt test:compile
-        - sbt it:compile
-        - sbt test
+        - sbt server/test:compile
+        # Compile integration tests to fail build early if they are broken
+        - sbt server/it:compile
+        # Run backend tests
+        - sbt server/test
     # Frontend tests
     - stage: test
-      install:
-        - cd webui
-        - yarn install
-        - yarn setup
       script:
-        - yarn format:validate
-        - yarn compile
-        - yarn test
+        - sbt webui/yarnFormatValidate
+        # Generate resources to ensure that webpack builds cleanly
+        - sbt webui/copyResources
```

Comment:
It copies all resources into the binary output directory, ie, `/target/scala-X.XX/classes/`, which implies running all resource generators, ie, webpack in our case.  In the end this triggers a webpack build from SBT.

It's the only way to trigger resources generators, as there's no dedicated command for this.  Note that `sbt compile` does not generate resources, because resources are not strictly required for compilation.  They only trigger when compiling across project dependencies, ie, `server/compile` will trigger resources generators of webui.

---

Try running `sbt webui/clean webui/copyResources`.  You'll see that SBT starts a webpack build after which the generated assets end up in `webui/target/scala-2.11/classes`.